src/DGtal/base/Common.h

```diff
@@ -111,7 +111,7 @@
 #include "DGtal/base/Exceptions.h"
 
 //////////////////////////////////////////////////////////////////////////////
-namespace DGtal
+namespaced DGtal
```

Comment:
What is the meaning of "namespaced" in C++ ?

Reply:
None, it is meant to trigger a compilation error to test Travis behaviour :p

Reply:
ok :)